asclepias_broker/harvester/tasks.py

```diff
@@ -16,27 +16,29 @@
 
 
 @shared_task(ignore_result=True, max_retries=3, default_retry_delay=10 * 60)
-def harvest_metadata_identifier(harvester: str, identifier: str, scheme: str):
+def harvest_metadata_identifier(harvester: str, identifier: str, scheme: str,
+                                providers: List[str]):
     """."""
     try:
         h = current_harvester.metadata_harvesters[harvester]
-        h.harvest(identifier, scheme)
+        h.harvest(identifier, scheme, providers)
     except Exception as exc:
         harvest_metadata_identifier.retry(exc=exc)
 
 
 @shared_task(ignore_result=True)
-def harvest_metadata(identifiers: Optional[List[Tuple[str, str]]],
+def harvest_metadata(identifiers: Optional[List[dict]],
                      eager: bool = False):
     """."""
     if identifiers:
         identifiers_to_harvest = ((i, v) for i, v in identifiers)
     else:  # use queue
         identifiers_to_harvest = current_harvester.metadata_queue.consume()
-    for value, scheme in identifiers_to_harvest:
+    for value, scheme, providers in identifiers_to_harvest:
```

Comment:
Since the payload format has changed to a `dict` you'll have to do something like:

```python
...
for payload in identifiers_to_harvest:
    value = payload['identifier']
    scheme = payload['scheme']
    providers = payload['scheme']
    ...
```

Make sure that the `identifiers_to_harvest` iterable actually ends up having this kind of data format